src/components/inputs/Input/Input.style.ts

```diff
@@ -52,15 +52,14 @@ export const HiddenMark = styled.input.attrs(({ type }) => ({
   ${fauxToggleChecked};
 `;
 
-function fauxMarkChecked({ theme, disabled }) {
+function fauxMarkChecked({ disabled }) {
   return css`
     &:checked ~ ${Faux} {
-      border: 1px solid
-        ${rgba(theme.content.color, disabled ? 0 : 0.75)};
+      border: 1px solid ${rgba(blue(500), disabled ? 0 : 1)};
```

Comment:
Does this change behavior in dark mode

Reply:
It doesn't, the the border remains consistent whether it is in dark or light mode and it looks like it'll be consistent with the SVG's border styling: https://github.com/vimeo/iris/blob/main/src/components/inputs/Input/Input.style.ts#L127.